index.html

```diff
@@ -5,184 +5,285 @@
-        <ul class="tag">
-            <li><button type="button">html</button></li>
-            <li><button type="button">css</button></li>
-            <li><button type="button">javaScript</button></li>
-        </ul>
-        <button class="see-project" type="button">See Project</button>
-    </section>
-    <section class="work card-4">
-        <img src="./img/Snapshoot-Portfolio4.png" alt="">
-        <h2>Multi-Post Stories</h2>
-        <ul class="frame">
-            <li class="client">CANOPY</li>
-            <li><img src="./img/Counter.png" alt=""></li>
-            <li>Back End Dev</li>
-            <li><img src="./img/Counter.png" alt=""></li>
-            <li>2015</li>
-        </ul>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        <ul class="tag">
-            <li><button type="button">html</button></li>
-            <li><button type="button">css</button></li>
-            <li><button type="button">javaScript</button></li>
-        </ul>
-        <button class="see-project" type="button">See Project</button>
     </section>
-    <section class="about">
-        <h1>About
-            Myself</h1>
-        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
-            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-            contact me.</p>
-        <h4>LET’S CONNECT</h4>
-        <div class="social-icons-2">
-            <div>
-                <a href="">
-                    <img src="./img/Vector3.png" alt="github icon">
-                </a>
+    <section id="portfolio" class="work">
+        <div class="grid-item grid-item1">
+            <div class="work-img work-img1">
+
+            </div>
+            <div class="description-container1">
+                <h2>Tonic</h2>
+                <div class="work-content work-content1">
+
+                    <p>CANCOPY</p>
+                    <ul>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="work-description">
+                    <p class="p-1">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="categories">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button" class="work-button">See Project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="grid-item grid-item2">
+            <div class="work-img work-img2">
+
```

Comment:
- [ ] As I mentioned in the previous comments, make sure you use the correct spacing(margin or padding) at the left and right and the correct **color** on the text for the whole page to make stick to the Figma template design.